src/psyclone/tests/domain/lfric/dofkern_test.py

```diff
@@ -281,40 +281,38 @@ def test_multi_invoke_cell_dof_builtin(tmpdir, monkeypatch, annexed, dist_mem):
     # generated
 
     # Use statements
-    output = (
-        "      USE testkern_mod, ONLY: testkern_code\n"
-        "      USE testkern_dofs_mod, ONLY: testkern_dofs_code\n"
-        )
+    assert "    use testkern_mod, only : testkern_code\n" in code
+    assert "    use testkern_dofs_mod, only : testkern_dofs_code\n" in code
     if dist_mem:
         # Check mesh_mod is added to use statements
-        output += ("      USE mesh_mod, ONLY: mesh_type\n")
-    assert output in code
+        assert "    use mesh_mod, only : mesh_type\n" in code
 
     # Consistent declarations
-    output = (
-        "      REAL(KIND=r_def), intent(in) :: scalar_arg, a\n"
-        "      TYPE(field_type), intent(in) :: f1, f2, f3, f4, m1, m2\n"
-        "      INTEGER(KIND=i_def) cell\n"
-        "      INTEGER(KIND=i_def) df\n"
-        "      INTEGER(KIND=i_def) loop2_start, loop2_stop\n"
-        "      INTEGER(KIND=i_def) loop1_start, loop1_stop\n"
-        "      INTEGER(KIND=i_def) loop0_start, loop0_stop\n"
-        "      INTEGER(KIND=i_def) nlayers_f1\n"
-        "      REAL(KIND=r_def), pointer, dimension(:) :: m2_data => null()\n"
-        "      REAL(KIND=r_def), pointer, dimension(:) :: m1_data => null()\n"
-        "      REAL(KIND=r_def), pointer, dimension(:) :: f4_data => null()\n"
-        "      REAL(KIND=r_def), pointer, dimension(:) :: f3_data => null()\n"
-        "      REAL(KIND=r_def), pointer, dimension(:) :: f2_data => null()\n"
-        "      REAL(KIND=r_def), pointer, dimension(:) :: f1_data => null()\n"
-        )
-    assert output in code
+    assert """
+    type(field_type), intent(in) :: f1
```

Comment:
It would be nice if we generated declarations grouped by type but that's definitely something for another time :-)